src/components/pages/my-space.less

```diff
@@ -11,3 +11,8 @@
 .MySpacePage-usedStorageLabel {
   margin-right: 15px;
 }
+
+.MySpacePage-usedStorageBar {
+  margin: 10px 0;
```

Comment:
It would be better to put this margin in the usage side of the component, cause I'm adding this component to the repository browser and it would create too much space 😬